### ISSUE TITLE
Breaking API-change introduced in Composer 1.1 branch

### ISSUE THREAD

**Issue by kghbln** [wikimedia/composer-merge-plugin#119]
Supposedly a breaking API-change was introduced in Composer 1.1 branch (Composer version 1.1.0 2016-05-10 15:21:19):
```
The "wikimedia/composer-merge-plugin" plugin was skipped because it requires a Plugin API version ("1.0.0") that does not match your Composer installation ("1.1.0"). You may need to run composer update with the "--no-plugins" option.

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - wikimedia/composer-merge-plugin v1.0.0 requires composer-plugin-api 1.0.0 -> no matching package found.
    - wikimedia/composer-merge-plugin v1.0.0 requires composer-plugin-api 1.0.0 -> no matching package found.
    - Installation request for wikimedia/composer-merge-plugin 1.0.0 -> satisfiable by wikimedia/composer-merge-plugin[v1.0.0].
```
I will downgrade to the Composer 1.0 branch which hopefully helps to avoid the issue for now.

**Comment by bd808** [comment]
This problem was fixed by c938491 which is included in composer-merge-plugin v1.2.0 and later. The project that you are using composer-merge-plugin with needs to update its dependency to require a newer version. For MediaWiki 1.25.x the needed fix was merged in https://gerrit.wikimedia.org/r/#/c/288349/ but is not included in any tarball release yet. See also https://phabricator.wikimedia.org/T135102

**Comment by kghbln** [comment]
Thanks a lot for your insight. Yeah, I expected this to be just a tiny issue.

> I will downgrade to the Composer 1.0 branch which hopefully helps to avoid the issue for now.

This helped me for the time being and I will stay on Composer 1.0.3 until the next MW 1.25 is out.
